### PR TITLE
[New] Update Drupal 8 on Distro X

### PR DESCRIPTION
**Pull request by leslitagordita**
* This work was originally drafted by @Rajakavitha1 in PR: https://github.com/linode/docs/pull/3148
* This new PR includes tech edits, rebase, and reorganization of sections after new changes brought in by the rebase.

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit cf4fdaa2bfb4e394a2f4ba17481481800e1c7a48

https://deploy-preview-3159--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-debian-10/index.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        cd /var/www/html/example.com
+        sudo wget https://ftp.drupal.org/files/projects/drupal-8.8.2.tar.gz
+
+## Upgrade Your Site
+
+Now that the Drupal core archive is saved to your Linode, you are ready to proceed with the version upgrade of your site.
+
+###  Enable Maintenance Mode
+
+Drupal's *maintenance mode* allows users with the right permissions to use your site while all anyone else will be presented with a message that the site is under maintenance.
```
Comment (andystevensname) [inline docs/websites/cms/drupal/how-to-update-drupal-8-on-debian-10/index.md]: Same as above, "everyone else"

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-ubuntu-18-04/index.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        cd /var/www/html/example.com
+        sudo wget https://ftp.drupal.org/files/projects/drupal-8.8.2.tar.gz
+
+## Upgrade Your Site
+
+Now that the Drupal core archive is saved to your Linode, you are ready to proceed with the version upgrade of your site.
+
+###  Enable Maintenance Mode
+
+Drupal's *maintenance mode* allows users with the right permissions to use your site while all anyone else will be presented with a message that the site is under maintenance.
```
Comment (andystevensname) [inline docs/websites/cms/drupal/how-to-update-drupal-8-on-ubuntu-18-04/index.md]: Same as above, "everyone else"

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-debian-10/index.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+1.  While logged into your Drupal site, navigate to **Configuration**. Under the **Development** heading, click on **Maintenance mode**.
+
+    ![Maintenance Mode](drupal-updates-maintenance.png)
```
Comment (andystevensname) [inline docs/websites/cms/drupal/how-to-update-drupal-8-on-debian-10/index.md]: Please outline the Maintenance Mode box or use an arrow to call it out.

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-ubuntu-18-04/index.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+1.  While logged into your Drupal site, navigate to **Configuration**. Under the **Development** heading, click on **Maintenance mode**.
+
+    ![Maintenance Mode](drupal-updates-maintenance.png)
```
Comment (andystevensname) [inline docs/websites/cms/drupal/how-to-update-drupal-8-on-ubuntu-18-04/index.md]: Outline Maintenance Mode or use an arrow to call it out.

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-debian-10/index.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+    {{< note >}}
+If **Available updates** is not listed, enable the [Update Manager](https://www.drupal.org/docs/8/core/modules/update-manager) module by navigating to the **Extend** menu item in the Admin Toolbar. See [Drupal's documentation](https://www.drupal.org/docs/8/extending-drupal-8/installing-drupal-8-modules#s-step-2-enable-the-module) for more details on enabling modules.
+{{< /note >}}
+
+1.  Right click the link under the **RECOMMENDED VERSION** heading and copy the link address and paste it somewhere you can access later.
+
+    ![A Drupal Update](drupal-updates-download.png)
+
+    {{< note >}}
+If you receive an error when your Drupal 8 installation checks for available updates, it may be having issues communicating with the Drupal website to see their are updates. You can check your site's recent log messages, by navigating to **Reports** and selecting **Recent log messages** to further investigate the issue.
```
Comment (andystevensname) [inline docs/websites/cms/drupal/how-to-update-drupal-8-on-debian-10/index.md]: if there

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-centos-8/index.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+    {{< note >}}
+If **Available updates** is not listed, enable the [Update Manager](https://www.drupal.org/docs/8/core/modules/update-manager) module by navigating to the **Extend** menu item in the Admin Toolbar. See [Drupal's documentation](https://www.drupal.org/docs/8/extending-drupal-8/installing-drupal-8-modules#s-step-2-enable-the-module) for more details on enabling modules.
+{{< /note >}}
+
+1.  Right click the link under the **RECOMMENDED VERSION** heading and copy the link address and paste it somewhere you can access later.
+
+    ![A Drupal Update](drupal-updates-download.png)
+
+    {{< note >}}
+If you receive an error when your Drupal 8 installation checks for available updates, it may be having issues communicating with the Drupal website to see their are updates. You can check your site's recent log messages, by navigating to **Reports** and selecting **Recent log messages** to further investigate the issue.
```
Comment (andystevensname) [inline docs/websites/cms/drupal/how-to-update-drupal-8-on-centos-8/index.md]: to see if there

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-ubuntu-18-04/index.md [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+    {{< note >}}
+If **Available updates** is not listed, enable the [Update Manager](https://www.drupal.org/docs/8/core/modules/update-manager) module by navigating to the **Extend** menu item in the Admin Toolbar. See [Drupal's documentation](https://www.drupal.org/docs/8/extending-drupal-8/installing-drupal-8-modules#s-step-2-enable-the-module) for more details on enabling modules.
+{{< /note >}}
+
+1.  Right click the link under the **RECOMMENDED VERSION** heading and copy the link address and paste it somewhere you can access later.
+
+    ![A Drupal Update](drupal-updates-download.png)
+
+    {{< note >}}
+If you receive an error when your Drupal 8 installation checks for available updates, it may be having issues communicating with the Drupal website to see their are updates. You can check your site's recent log messages, by navigating to **Reports** and selecting **Recent log messages** to further investigate the issue.
```
Comment (andystevensname) [inline docs/websites/cms/drupal/how-to-update-drupal-8-on-ubuntu-18-04/index.md]: to see if there are

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-centos-8/index.md [hunk 8]
```diff
@@ -0,0 +1,141 @@
+---
+author:
+    name: Linode
+    email: docs@linode.com
+description: 'This guide will show you how to update your Drupal 8 installation running on a CentOS 8 Linode.'
```
Comment (andystevensname) [inline docs/websites/cms/drupal/how-to-update-drupal-8-on-centos-8/index.md]: needs an og_description

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-debian-10/index.md [hunk 9]
```diff
@@ -0,0 +1,133 @@
+---
+author:
+    name: Linode
+    email: docs@linode.com
+description: 'This guide will show you how to update your Drupal 8 installation running on a Debian 10 Linode.'
```
Comment (andystevensname) [inline docs/websites/cms/drupal/how-to-update-drupal-8-on-debian-10/index.md]: needs an og_description

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-ubuntu-18-04/index.md [hunk 10]
```diff
@@ -0,0 +1,137 @@
+---
+author:
+    name: Linode
+    email: docs@linode.com
+description: 'This guide will show you how to update your Drupal 8 installation running on an Ubuntu 18.04 Linode.'
```
Comment (andystevensname) [inline docs/websites/cms/drupal/how-to-update-drupal-8-on-ubuntu-18-04/index.md]: needs an og_description

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-centos-8/index.md [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+        cd /var/www/html/example.com
+        sudo wget https://ftp.drupal.org/files/projects/drupal-8.8.2.tar.gz
+
+## Upgrade Your Site
+
+Now that the Drupal core archive is saved to your Linode, you are ready to proceed with the version upgrade of your site.
+
+###  Enable Maintenance Mode
+
+Drupal's *maintenance mode* allows users with the right permissions to use your site while all anyone else will be presented with a message that the site is under maintenance.
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/how-to-update-drupal-8-on-centos-8/index.md]: fixed

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-centos-8/index.md [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+
+1.  While logged into your Drupal site, navigate to **Configuration**. Under the **Development** heading, click on **Maintenance mode**.
+
+    ![Maintenance Mode](drupal-updates-maintenance.png)
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/how-to-update-drupal-8-on-centos-8/index.md]: fixed

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-debian-10/index.md (hunk 1 above)
Comment (leslitagordita) [reply docs/websites/cms/drupal/how-to-update-drupal-8-on-debian-10/index.md]: fixed

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-ubuntu-18-04/index.md (hunk 2 above)
Comment (leslitagordita) [reply docs/websites/cms/drupal/how-to-update-drupal-8-on-ubuntu-18-04/index.md]: fixed

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-debian-10/index.md (hunk 3 above)
Comment (leslitagordita) [reply docs/websites/cms/drupal/how-to-update-drupal-8-on-debian-10/index.md]: fixed

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-ubuntu-18-04/index.md (hunk 4 above)
Comment (leslitagordita) [reply docs/websites/cms/drupal/how-to-update-drupal-8-on-ubuntu-18-04/index.md]: fised

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-debian-10/index.md (hunk 5 above)
Comment (leslitagordita) [reply docs/websites/cms/drupal/how-to-update-drupal-8-on-debian-10/index.md]: fixed

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-centos-8/index.md (hunk 6 above)
Comment (leslitagordita) [reply docs/websites/cms/drupal/how-to-update-drupal-8-on-centos-8/index.md]: fixed

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-ubuntu-18-04/index.md (hunk 7 above)
Comment (leslitagordita) [reply docs/websites/cms/drupal/how-to-update-drupal-8-on-ubuntu-18-04/index.md]: fixed

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-centos-8/index.md (hunk 8 above)
Comment (leslitagordita) [reply docs/websites/cms/drupal/how-to-update-drupal-8-on-centos-8/index.md]: fixed

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-debian-10/index.md (hunk 9 above)
Comment (leslitagordita) [reply docs/websites/cms/drupal/how-to-update-drupal-8-on-debian-10/index.md]: fixed

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-ubuntu-18-04/index.md (hunk 10 above)
Comment (leslitagordita) [reply docs/websites/cms/drupal/how-to-update-drupal-8-on-ubuntu-18-04/index.md]: fixed

File: docs/websites/cms/drupal/how-to-update-drupal-8-on-centos-8/index.md [hunk 13]
```diff
@@ -0,0 +1,142 @@
...
+
+1. Create an archive of your existing site files and move it into the `backups` directory. Ensure you replace `example.com` with your own site's domain name.
+
+        cd /var/www/html/example.com/public_html
+        sudo tar -cvzf example.com-BCKP-$(date +%Y%m%d).tar.gz ./
+        sudo mv -v example.com-BCKP-*.tar.gz ../backups
+
+    {{< note >}}
+This process can also be scripted and run on a regular basis using [cron](/docs/tools-reference/tools/schedule-tasks-with-cron/).
+    {{</ note >}}
```
Comment (Rajakavitha1) [inline docs/websites/cms/drupal/how-to-update-drupal-8-on-centos-8/index.md]: You can also schedule backups with the 'backup_migrate' module in the Drupal UI as well.

Comment (andystevensname) [reply docs/websites/cms/drupal/how-to-update-drupal-8-on-centos-8/index.md]: Looks like that module has a number of outstanding bugs, I think for this guide we can leave it out.

**Pull request merged**